### ISSUE TITLE
[DEMO] Iris Classification using Qnode, Keras Optimizer and Loss function.

### ISSUE THREAD

**Issue by gahanh** [PennyLaneAI/qml#156]
#### General information

**Name**
Hemant Gahankari

**Affiliation** (optional)
None
--------------------------------------------------------------------------------

#### Demo information

**Title**
Iris Classification using Qnode and Keras Optimizer and Loss function.

**Abstract**
This example is created to explain how to create a quantum function and train a quantum function using keras optimizer directly. i.e. not using Keras Layer.

Objective is to train a quantum function to predict classes of Iris dataset.

**Relevant links**
https://colab.research.google.com/drive/17Qri3jUBpjjkhmO6ZZZNXwm511svSVPw?usp=sharing

**Comment by antalszava** [comment]
The demo was added to the [community demos page](https://pennylane.ai/qml/demos_community.html).
